containers/datalab/run.sh

```diff
@@ -49,4 +49,5 @@ docker run -it --entrypoint=$ENTRYPOINT \
   -v "$CONTENT:/content" \
   -e "PROJECT_ID=$PROJECT_ID" \
   -e "DATALAB_ENV=local" \
+  -e "KG_URL=${KG_URL}" \
```

Comment:
Should this environment variable be added to `containers/datalab/run-extended.sh` also ? https://github.com/GoogleCloudPlatform/datalab/blob/master/containers/datalab/run-extended.sh